vulnerabilities/views.py

```diff
@@ -62,25 +64,21 @@ def get_purl_version_class(purl: models.Package):
     return purl_version_class
 
 
-class PackageSearch(ListView):
+class PackageSearch(PaginatedListViewMixin, ListView):
```

Comment:
Add the PaginatedListViewMixin in it and this would convert any ListView into paginated class.